code/game/objects/items/tcg/tcg.dm

```diff
@@ -526,9 +526,9 @@ GLOBAL_LIST_EMPTY(tcgcard_radial_choices)
 				toReturn += "[pack.type] has a rarity [template.rarity] on the card [template.id] that does not exist\n"
 				continue
 		//Lets run a check to see if all the rarities exist that we want to exist exist
-		for(var/I in pack.rarity_table)
-			if(!GLOB.cached_cards[pack.series][I])
-				toReturn += "[pack.type] does not have the required rarity [I]\n"
+		for(var/_rarity in pack.rarity_table)
```

Comment:
Why the underscore prefix?

Reply:
rarity is a var on `src` and i couldn't think of anything else to use there that would make sense

Reply:
Fair, what about `pack_rarity`?

Just `rarity` *should* also work, IIRC, but I understand not wanting to repeat it.

Reply:
works for me